dashboards/key_metrics/views/lp.py

```diff
@@ -73,6 +87,18 @@ def fetch_data(date_range, chain):
     title="TVL",
     color="label",
 )
+chart_core_account_activity_daily = chart_lines(
+    data["core_account_activity_daily"]
+    .groupby(["date", "action"])
+    .nof_accounts.sum()
+    .reset_index(),
```

Comment:
we should keep all logic like this in the `fetch_data` function, since the data gets cached.